helmfile/values/gatekeeper/gatekeeper.yaml.gotmpl

```diff
@@ -1,7 +1,17 @@
 validatingWebhookFailurePolicy: Fail
 validatingWebhookTimeoutSeconds: {{ .Values.opa.validatingWebhookTimeoutSeconds }}
+validatingWebhookExemptNamespacesLabels:
+  kubernetes.io/metadata.name:
```

Comment:
It shouldn't be possible to modify this label of a namespace, so customers shouldn't be able to abuse it.